library/ajax/drug_autocomplete/index.php

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Demo</title>
+  <link rel="stylesheet" href="../../library/js/jquery-ui.min.css" type="text/css" /> 
+</head>
+<body> 
+
+	<form action='' method='post'>
+		<p><label>Names:</label><input type='text' name='names' value='' class='auto'></p>
+	</form>
+
+<script type="text/javascript" src="../../library/js/jquery-1.9.1.min.js"></script>
```

Comment:
This file is not used in the program. It is an example of how the code works. I can remove it with no consequences.